test/plugins/camelCaseMotion.test.ts

```diff
@@ -0,0 +1,295 @@
+import { Globals } from '../../src/globals';
+import { getTestingFunctions } from '../testSimplifier';
+import { cleanUpWorkspace, setupWorkspace, reloadConfiguration } from './../testUtils';
+
+suite('camelCaseMotion plugin', () => {
+  const { newTest } = getTestingFunctions();
+
+  setup(async () => {
+    await setupWorkspace();
```

Comment:
Can you setup the config in this manner instead:

```
    let configuration = new Configuration();
    //whatever you need to update to configuration
    await setupWorkspace(configuration);
```

Saves us from having to reload the config.